src/include/threadpool/mono_queue_pool.h

```diff
@@ -24,7 +24,9 @@ namespace threadpool {
  */
 class MonoQueuePool {
  public:
-  MonoQueuePool(int task_queue_size = 32, int worker_pool_size = 4)
+  MonoQueuePool() = delete;
+
+  MonoQueuePool(int task_queue_size, int worker_pool_size)
```

Comment:
These arguments probably shouldn't be signed, right? Can you change them to `uint32_t`?